demo/src/docs/Home.js

```diff
@@ -6,12 +6,7 @@ import Button from "elevate-ui/Button";
 const Home = ({ classes }) => (
   <div className={classes.container}>
     <Typography type="title">Welcome</Typography>
-    <Typography
-      type="body"
-      style={{
-        margin: "8px 0 16px",
-      }}
-    >
+    <Typography type="body" className={classes.body}>
       Elevate-UI is an opinionated toolkit of React components and patterns used
       for the rapid prototyping and development of web apps and sites for Elm
       Street Technology. It seeks to establish best practices around
```

Comment:
Also, while its not in this diff, can we add a proper color to the link here? It's purple when visited, etc...